src/PlaywrightSharp/Helpers/StringExtensions.cs

```diff
@@ -64,33 +64,53 @@ public static string UnQuote(this string value)
 
             var result = new Dictionary<string, string>();
 
-            foreach (string keyvalue in query.Split('&'))
+            foreach (string keyValue in query.Split('&'))
             {
-                string[] pair = keyvalue.Split('=');
+                string[] pair = keyValue.Split('=');
                 result[pair[0]] = pair[1];
             }
 
             return result;
         }
 
         /// <summary>
-        /// Determin if the script is a javascript function and not an expression.
+        /// Determine if the script is a javascript function and not an expression.
         /// </summary>
         /// <param name="script">Script to evaluate.</param>
         /// <returns>Whether the script is a function or not.</returns>
-        public static bool IsJavascriptFunction(this string script)
+        public static bool IsJavascriptFunction(ref string script)

```

Comment:
why not use the same flow as playwright does?

Reply:
Thanks for pushing here. It turns out that pw adds the parenthesis to test the function, but we don't need to send that function with parenthesis.

Reverted.